symfony/plugins/orangehrmLeavePlugin/Report/LeaveTypeLeaveEntitlementUsageReport.php

```diff
@@ -69,7 +71,7 @@ public function getHeaderDefinition(): Header
     {
         return new Header(
             [
-                (new Column(self::PARAMETER_EMPLOYEE_NAME))->setName('Employee')
+                (new Column(self::PARAMETER_EMPLOYEE_NAME))->setName($this->getI18NHelper()->trans('general.employee'))
                     ->setPin(Column::PIN_COL_START)
                     ->setSize(self::DEFAULT_COLUMN_SIZE),
                 (new Column(self::PARAMETER_ENTITLEMENT_DAYS))->setName('Leave Entitlements (Days)')
```

Comment:
I assume we can translate the others later right? like Leave Entitlements (Days)

Reply:
yes